### DISCUSSION TITLE
posts/seguridad-activa-control-redes/

### DISCUSSION THREAD

**Discussion by giscus** [marcosruiz/marcosruiz.github.io#4] [General]
# posts/seguridad-activa-control-redes/

Espiar nuestra red

https://marcosruiz.github.io/posts/seguridad-activa-control-redes/

**Comment by DonViito10** [comment]
Demasiado texto...

**Reply by rtapiamateos** [reply]
Si, la verdad...

**Reply by marcosruiz** [reply]
No has leido tanto en tu vida hulio xD

**Reply by rtapiamateos** [reply]
leído lleva tilde.
